src/containers/Profile/index.js

```diff
@@ -109,6 +115,24 @@ class ProfileComponent extends React.Component {
     ) {
       ReactDOM.findDOMNode(this.explorerBody).scrollIntoView(false);
     }
+
+    if (prevProps.reviewsLoading && !reviewsLoading && !networkSwitchSet) {
+      this.setState({
+        networkSwitchSet: true
+      });
+
+      const { reviews, fulfiller, issuer } = queryStringToObject(
+        location.search
+      );
+
+      if (!issuer && (fulfiller || reviewCount === 0)) {
+        setActiveNetworkSwitch('fulfiller');
+      }
+
+      if (reviews) {
+        setReviewsModalVisible(true);
```

Comment:
@codeluggage do you think it is weird to have this querystring but the fact that it doesn't disappear if they click to the other tab?

Reply:
@villanuevawill Yes, that is weird! I don't like it! Didn't think about it at all until you brought it up - it would be much better if it was dynamic so you could cleanly share the URL so people can go directly to either issuer or fulfiller reviews for a user. 

Worth doing it in this PR, or merge this and start a new one?